### PR TITLE
Add option to enable PvP by default

### PR DESCRIPTION
**Pull request by luk3yx**

### CODE REVIEW

File: settingtypes.txt [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # Static paths do not work well with settings
 #areas.filename (Configuration file path) string (world_path)/areas.dat
 
+# When enabled, makes PvP opt-out rather than opt-in.
```
Comment (MoNTE48) [inline settingtypes.txt]: Can the wording be easier for better understanding by non-English-speaking players?

Comment (luk3yx) [reply settingtypes.txt]: I could make it simpler but I feel like it'd lose some nuance and not be completely accurate.

Maybe "Turn PvP on everywhere" would be simple enough? But it doesn't explain how to turn it back off, I can't figure out how to succinctly explain it without using "opt-out" or some similar word

Comment (MoNTE48) [reply settingtypes.txt]: I believe that  "Turn PvP on everywhere" is most understandable

**Comment by MoNTE48** [comment]
Can you update `settingtypes.txt` ?

**Comment by luk3yx** [comment]
This PR already adds the new setting to settingtypes.txt?

```
# Turn on PvP everywhere
areas.pvp_by_default (PvP by default) bool false
```

**Pull request merged**